common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -640,25 +670,39 @@ def available_translations(self, transcripts, verify_assets=None):
 
         # If we've gotten this far, we're going to verify that the transcripts
         # being referenced are actually in the contentstore.
+        val_transcript_languages = []
+        if verify_val_transcripts:
```

Comment:
What's the purpose of this verify ? I think i got what you're trying to do here but not sure about the name of this method. I think it's not verifying anything except fallback to val codes.